src/sentry/static/sentry/app/components/buttons/button.jsx

```diff
@@ -1,12 +1,22 @@
+import {Flex, Box} from 'grid-emotion';
 import {Link} from 'react-router';
 import PropTypes from 'prop-types';
 import React from 'react';
+import styled from 'react-emotion';
 import classNames from 'classnames';
 
-import FlowLayout from '../flowLayout';
-
+import InlineSvg from '../inlineSvg';
 import '../../../less/components/button.less';
 
+const Icon = styled(Box)`
+  margin-right: ${p => (p.size ? '6px' : '8px')};
```

Comment:
When would size be `false`y here?

Reply:
Good catch, it should be `p.size == 'small'`. Pushing a fix.